### PR TITLE
docs: refine cli commands

### PR DESCRIPTION
**Pull request by forcodedancing**
docs for cli commands

### CODE REVIEW

File: docs/api/blockchain-cli/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ aspects of the commands, such as gas prices or which node to broadcast to.
 Here is an example of a command a user might enter to interact with `gnfd` in order to send some tokens:
 
 ```bash
-$ gnfd tx bank send $MY_VALIDATOR_ADDRESS $RECIPIENT 1000BNB --gas auto
+gnfd tx bank send $MY_VALIDATOR_ADDRESS $RECIPIENT 1000BNB --gas auto
```
Comment (KeefeL) [inline docs/api/blockchain-cli/README.md:25]: Better not use $MY_VALIDATOR_ADDRESS, use $MY_ADDRESS instead?

File: docs/api/blockchain-cli/challenge.md [hunk 2]
```diff
@@ -12,10 +12,43 @@ Users can submit a challenge and query the latest attested challenges through cl
...
+
+```shell
+gnfd tx challenge submit ${sp-operator-address} ${bucket-name} ${object-name} ${random-index} ${segment-index} --from ${key} --node ${node} -y
+```
+
+${key} is the name of local key.
+
+${bucket-name} and ${object-name} specify which object you want to submit challenge for.
+
+${random-index} ${segment-index} specify which segment of the object you want to submit challenge for. If you do not know the index, you and set ${random-index} to `true`.
```
Comment (KeefeL) [inline docs/api/blockchain-cli/challenge.md:27]: ${random-index} and ${segment-index}

you and set -> you can set?

File: docs/api/blockchain-cli/challenge.md [hunk 3]
```diff
@@ -26,21 +59,57 @@ A user can query and interact with the `challenge` module using the CLI.
...
+  slash_amount_min: "10000000000000000"
+  slash_amount_size_rate: "0.008500000000000000"
+  slash_cooling_off_period: "300"
+  sp_slash_counting_window: "43200"
+  sp_slash_max_amount: "500000000000000000"
 ```
 
 #### latest-attested-challenges  
 
 The `latest-attested-challenges` command allows users to query the latest challenges that have been attested by the user.
```
Comment (KeefeL) [inline docs/api/blockchain-cli/challenge.md:101]: I think this should be attested by the validators? Not by the user?

File: docs/api/blockchain-cli/challenge.md [hunk 4]
```diff
@@ -52,91 +121,81 @@ Example Output:
    result: CHALLENGE_SUCCEED
 ```
 
-#### inturn-attestation-submitter
+#### attested-challenge
 
-The `inturn-attestation-submitter` command allows users to query the off-chain challenger service that is currently in charge of attesting.   
+The `attested-challenge` command allows users to query a specific challenge that have been attested by the user.
```
Comment (KeefeL) [inline docs/api/blockchain-cli/challenge.md:126]: have been attested by the vadiators?

File: docs/api/blockchain-cli/challenge.md [hunk 5]
```diff
@@ -52,91 +121,81 @@ Example Output:
...
+
+Be noted, only the latest `attestation_kept_count` challenged will be used to serve this query.
```
Comment (KeefeL) [inline docs/api/blockchain-cli/challenge.md:128]: Only the challenge within the latest `attestation_kept_count` challenges will be used to serve this query?

File: docs/api/blockchain-cli/challenge.md [hunk 6]
```diff
@@ -52,91 +121,81 @@ Example Output:
...
 Usually the challenge will be handled within minutes, and the result of the challenge can be queried using the following command. 
 However, be noted, the result is not kept forever, and only the latest `attestation_kept_count` challenge results will be kept.
+
 ```shell
- ./gnfd query challenge latest-attested-challenges --node https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443 
+gnfd query challenge latest-attested-challenges --node https://greenfield-chain.bnbchain.org:443 
 ```
 
-The result of the challenge can be queried using the `AttestedChallenge`, `LatestAttestedChallenges` methods through [GRPC swagger](https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org/openapi).
+The result of the challenge can be queried using the `AttestedChallenge`, `LatestAttestedChallenges` methods through [GRPC swagger](https://greenfield-chain.bnbchain.org/openapi).
```
Comment (KeefeL) [inline docs/api/blockchain-cli/challenge.md:201]: The result of the challenge can be queried by the above query commands, or using the ....

File: docs/api/blockchain-cli/governance.md [hunk 7]
```diff
@@ -4,37 +4,96 @@ order: 7
 ---
 # On-chain Governance
 
+## Abstract
+
+In Greenfield, users can submit proposals for on-chain governance. 
+These proposals can include updates to blockchain parameters or cross-chain configurations, 
+or execution of some messages (e.g., create storage provider). 
+Once a proposal is submitted, it must be accompanied by a deposit that exceeds 
+the `Min Deposit` parameter (which is 10BNB on mainnet).
```
Comment (KeefeL) [inline docs/api/blockchain-cli/governance.md:13]: Is it true? I think this is 1BNB or 1000000000000000000BNB?

File: docs/api/blockchain-cli/validator-staking.md [hunk 8]
```diff
@@ -5,36 +5,208 @@ Order: 8
...
-## try self delegate 100BNB for validator0
-$ gnfd tx staking delegate $val 100000000000000000000BNB --from validator0 --home ./deployment/localup/.local/validator0 --keyring-backend test --node http://localhost:26750 -b block  -y
-## try unbond 100BNB from validator0
-$ gnfd tx staking unbond $val 100000000000000000000BNB --from validator0 --home ./deployment/localup/.local/validator0 --keyring-backend test --node http://localhost:26750 -b block  -y
-## query the unbonding-delegation
-$ gnfd q staking unbonding-delegation $val $val --node tcp://127.0.0.1:26750
+
+${key} is the name of local key.
+
+${grantee} specifies the address of the grantee. In this case, it is `0x7b5Fe22B5446f7C62Ea27B8BD71CeF94e03f3dF2`.
```
Comment (KeefeL) [inline docs/api/blockchain-cli/validator-staking.md:32]: it should be the gov module address: `0x7b5Fe22B5446f7C62Ea27B8BD71CeF94e03f3dF2`.

File: docs/api/blockchain-cli/validator-staking.md [hunk 9]
```diff
@@ -5,36 +5,208 @@ Order: 8
...
-## query the validator info
-$ gnfd q staking validator $val --node tcp://127.0.0.1:26750
-## try self delegate 100BNB for validator0
-$ gnfd tx staking delegate $val 100000000000000000000BNB --from validator0 --home ./deployment/localup/.local/validator0 --keyring-backend test --node http://localhost:26750 -b block  -y
-## try unbond 100BNB from validator0
-$ gnfd tx staking unbond $val 100000000000000000000BNB --from validator0 --home ./deployment/localup/.local/validator0 --keyring-backend test --node http://localhost:26750 -b block  -y
-## query the unbonding-delegation
-$ gnfd q staking unbonding-delegation $val $val --node tcp://127.0.0.1:26750
+
+${key} is the name of local key.
```
Comment (KeefeL) [inline docs/api/blockchain-cli/validator-staking.md:30]: ${key} is the name of the self delegator account's local key.

File: docs/api/blockchain-cli/validator-staking.md [hunk 10]
```diff
@@ -5,36 +5,208 @@ Order: 8
...
+    }
+  ],
+  "metadata": "4pIMOgIGx1vZGU=",
+  "deposit": "1000000000000000000BNB"
+}
+```
+
+```sh
+gnfd tx gov submit-proposal create-validator.json --from mykey --gas auto
+```
```
Comment (KeefeL) [inline docs/api/blockchain-cli/validator-staking.md:121]: Add a new part:

#### Create a new validator in one step

You can also use the `create-validator` command to create a new validator in one step, this command will include the above two steps.

```sh
gnfd tx staking create-validator create-validator.json --from ${self-delegator-key} --node ${node}
```

File: docs/api/blockchain-cli/payment.md [hunk 11]
```diff
@@ -58,156 +99,161 @@ A user can query and interact with the `payment` module using the CLI.
 
 The `query` commands allow users to query `payment` state.
 
+#### params
+
+The `params` command allows users to query the params of the `payment` module.
+
+```sh
+gnfd query bridge params [flags]
```
Comment (KeefeL) [inline docs/api/blockchain-cli/payment.md:107]: gnfd query payment params

File: docs/api/blockchain-cli/payment.md [hunk 12]
```diff
@@ -58,156 +99,161 @@ A user can query and interact with the `payment` module using the CLI.
...
+```
+
+Example:
+
+```sh
+gnfd query bridge params --node https://greenfield-chain.bnbchain.org:443
```
Comment (KeefeL) [inline docs/api/blockchain-cli/payment.md:113]: payment not bridge

File: docs/api/blockchain-cli/payment.md (hunk 11 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/payment.md:107]: fixed

File: docs/api/blockchain-cli/payment.md (hunk 12 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/payment.md:113]: fixed

File: docs/api/blockchain-cli/README.md (hunk 1 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/README.md:25]: yes, you are right

File: docs/api/blockchain-cli/challenge.md (hunk 3 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/challenge.md:101]: fixed

File: docs/api/blockchain-cli/challenge.md (hunk 4 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/challenge.md:126]: fixed

File: docs/api/blockchain-cli/challenge.md (hunk 2 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/challenge.md:27]: yes, a typo

File: docs/api/blockchain-cli/challenge.md (hunk 5 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/challenge.md:128]: Yes. Old challenge results are pruned.
Rephase the sentence.

File: docs/api/blockchain-cli/challenge.md (hunk 6 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/challenge.md:201]: updated

File: docs/api/blockchain-cli/governance.md (hunk 7 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/governance.md:13]: checked, it is 1BNB.
fixed

File: docs/api/blockchain-cli/validator-staking.md (hunk 8 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/validator-staking.md:32]: updated

File: docs/api/blockchain-cli/validator-staking.md (hunk 9 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/validator-staking.md:30]: updated

File: docs/api/blockchain-cli/validator-staking.md (hunk 10 above)
Comment (forcodedancing) [reply docs/api/blockchain-cli/validator-staking.md:121]: updated

**Review by KeefeL — APPROVED** [review]

**Pull request merged**